.env.example

```
@@ -26,6 +26,9 @@ AIRFLOW_SIMPLE_ADMIN_PASSWORD=admin
 AIRFLOW_SIMPLE_VIEWER_PASSWORD=viewer
 PIPELINE_BACKEND_BASE_URL=http://backend:8080
 PIPELINE_ARTIFACT_ROOT=/opt/airflow/artifacts
+PIPELINE_CALLBACK_ENABLED=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PIPELINE_CALLBACK_ENABLED key should go before the STORAGE_S3_ACCESS_KEY key

(UnorderedKey)